packages/button/Button.tsx

```diff
@@ -1,33 +1,44 @@
-import { h, Component, prop } from 'skatejs';
+import { h, Component, prop, ComponentProps } from 'skatejs';
 
-import { ColorType, cssClassForColorType, css } from '../_helpers';
+import {
+  ColorType,
```

Comment:
Just thinking - what about change order of imports to first helpers, then types? like:
`import {
 cssClassForColorType, 
 css, 
 ColorType, 
 IntrinsicCustomElement, 
 IntrinsicBoreElement, 
 ClickEvent 
} from '../_helpers'`